docs/topics/install-olympia/installation.rst

```diff
@@ -181,14 +181,30 @@ git.
 Database
 --------
 
-The database is initialized automatically using the `make full_init` command
-you saw earlier. If you want to start from scratch and recreate the database,
-you can just run the `make initialize_db` command.
+First of all, you'll need to create a database named ``olympia`` and grant
+permissions to the user, based on your database settings in your
+``local_settings.py``::
```

Comment:
This is correct, but I'd like a note specifying that it's optional. By default, the database used will be `olympia`, with the `root` user (without any password): https://github.com/mozilla/olympia/blob/master/lib/settings_base.py#L86

I'm not sure, but I think mysql installs by default with a passwordless root user, so you shouldn't have anything special to do, apart from creating the `olympia` database.

I would thus split this explanation in two parts:
1/ here's how to create the `olympia` database
2/ here's how to customize the user if you don't want to use the `root` user (with a more detailed explanation of how to specify the database url in the `local_settings.py` file)